dotnet/src/Connectors/Connectors.AssemblyAI/Services/AssemblyAIAudioToTextService.cs

```diff
@@ -57,17 +53,27 @@ public async Task<IReadOnlyList<TextContent>> GetTextContentsAsync(
     {
         Verify.NotNull(content);
 
+        if (executionSettings?.ExtensionData is not null && executionSettings.ExtensionData.Count > 0)
```

Comment:
Would be nice (if possible) support generic ExtensionData instead throwing exception.

Reply:
I agree, unfortunately, we cannot send arbitrary JSON via the SDK, but we could convert the params into JSON and the extension data, merge the JSON objects, and then  deserialize it back to the params class.